configuration/blockly/rules-blockly-persistence.md

```diff
@@ -34,40 +34,49 @@ More about that topic can be viewed at ![youtube](../images/blockly/youtube-logo
 
 ![statistical-value](../images/blockly/blockly-persistence-get-statistical-value.png)
 
-_Function:_ computes any of the below functions for the given item since the time provided by _ZonedDateTime_-Block
+_Function:_ computes any of the below functions for the given item since the time provided by _ZonedDateTime_-Block, until the time provided by _ZonedDateTime_-Block, or between 2 timepoints provided by _ZonedDateTime_-Blocks.
```

Comment:
Can you add an image like this that demonstrated "between"?

<img width="467" alt="image" src="https://github.com/user-attachments/assets/57f05693-7fcd-4dc2-be59-818fc5c448e3" />